h2o-algos/src/main/java/hex/ensemble/StackedEnsemble.java

```diff
@@ -71,6 +73,28 @@ public void init(boolean expensive) {
     super.init(expensive);
 
     checkFoldColumnPresent(_parms._metalearner_fold_column, train(), valid(), _parms.blending());
+    validateAndExpandBaseModels();
+  }
+
+  /**
+   * Validates base models and if grid is provided instead of a model it gets expanded.
+   */
+  private void validateAndExpandBaseModels() {
+    List<Key> base_models = new ArrayList<Key>();
```

Comment:
welcome to the mess of H2O naming "conventions" in Java.
Will try to sum it up:
1. use camelCase by default: local variables, method parameters, attributes, methods...
2. use snake_case for the fields exposed in Schemas (REST API uses snake_case).
3. fields in the POJOs mapped to the schema classe may use snake_case (this is because we use Java reflection by default to fill Schemas from POJOs and vice versa).
4. class attributes (and only those) are prefixed with underscore (except in Schemas, as it is the REST API).
5. test methods can be written camelCase or snake_case, this is because it is mush easier to read `test_stackedensemble_using_blending_with_invalid_model` than its camelCase equivalent..

Reply:
This is very helpful!